docs/APIs/security/tls.md

```diff
@@ -2,22 +2,22 @@
 
 mbed TLS provides a comprehensive SSL/TLS solution and makes it easy for developers to include cryptographic and SSL/TLS capabilities in their software and embedded products. As an SSL library it provides an intuitive API, readable source code and a minimal and highly configurable code footprint.
 
-<span class="notes">_**Note:** mbed TLS needs a secure source of random numbers, make sure that your target board has one and that it is fully ported to mbed OS. You can read more about this in our [porting guide](../../porting/tls_porting.md)._</span>
+<span class="notes">_**Note:** mbed TLS needs a secure source of random numbers; make sure that your target board has one and that it is fully ported to mbed OS. You can read more about this in our [porting guide](https://docs.mbed.com/docs/mbed-os-handbook/en/5.3/advanced/porting_guide/)._</span>
```

Comment:
At this point we can use https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/porting_guide/